sdk/application.go

```diff
@@ -12,14 +12,13 @@ type Repository struct {
 	Hook bool
 }
 
-// Application represent an application in a project
+// Application represent an application in a project.
 type Application struct {
 	ID                   int64                        `json:"id" db:"id"`
 	Name                 string                       `json:"name" db:"name" cli:"name,key"`
 	Description          string                       `json:"description"  db:"description"`
 	Icon                 string                       `json:"icon"  db:"icon"`
 	ProjectID            int64                        `json:"-" db:"project_id"`
-	ProjectKey           string                       `json:"project_key" db:"-" cli:"project_key"`
```

Comment:
We have a lot of entities with `project_key` - I'm pretty sure workflow have one.
The idea was to return an entity that contains all the data needed to make a call on the rest API.

We have to be careful with this kind of modification, because it may broke all the stuff out of the CDS perimeter that relies on it

Reply:
Rollback this.

Reply:
Change reverted.